onnxruntime/core/providers/cpu/tensor/upsample.cc

```diff
@@ -29,33 +29,6 @@ ONNX_CPU_OPERATOR_VERSIONED_TYPED_KERNEL(
     KernelDefBuilder().TypeConstraint("T", DataTypeImpl::GetTensorType<uint8_t>()),
     Upsample<uint8_t>);
 
-template <typename T>
```

Comment:
IMO, this should be kept unless the current implementation is not worse than this one. Otherwise, it introduces a perf regression.

Reply:
added back